picard/util/__init__.py

```diff
@@ -290,19 +291,60 @@ def find_existing_path(path):
     return decode_filename(path)
 
 
+def _add_windows_executable_extension(*executables):
+    processed_executables = []
+    for e in executables:
+        if ".py" in e:
+            processed_executables.append(e)
+            continue
+        if ".exe" in e:
```

Comment:
Rather use `e.endswith('.py')` and `e.endswith('.exe')`?

Reply:
Way better. :D